app/views/reports/costs.html.erb

```diff
@@ -17,13 +17,23 @@
         </div>
         <%= f.submit 'Search', class: 'btn btn-default t-search' %>
       <% end %>
+      <%= form_for @costs, url: reports_cost_breakdowns_path(format: :csv), method: :get, html: { class: 'inline form-inline' } do |f| %>
+        <%= f.hidden_field :start_month %>
+        <%= f.hidden_field :end_month %>
+        <%= f.submit 'Export Breakdown CSV', class: 'btn btn-default t-export-csv' %>
+      <% end %>
+      <%= form_for @costs, url: reports_cost_breakdowns_raw_path(format: :csv), method: :get, html: { class: 'inline form-inline' } do |f| %>
```

Comment:
painfully enough yes.. :(

`inline` to get the elements in the form on the same line and `form-inline` to get the separate CSV download forms on the same line.